examples/react-quickstart/src/components/Inbox.tsx

```diff
@@ -43,7 +44,7 @@ export const Inbox: React.FC = () => {
     <div className="Inbox">
       <div className="InboxHeader">
         <div className="InboxHeader__xmtp">
-          <img src="/xmtp-icon.png" alt="XMTP logo" width="32" />
+          <img src={xmtpIconUrl} alt="XMTP logo" width="32" />
```

Comment:
Hard coding bare URLs like this doesn't work in Reflame, because we use content-addressed urls (of the format `/xmtp-icon.png?~r_rid=some-long-id`) for every resource to maximize serving performance and simplify caching by allowing us to cache every resource practically forever at every layer without worrying about invalidation.

Importing the resource will give us a string containing the content-addressed URL.